.readthedocs.yaml

```diff
@@ -0,0 +1,12 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+python:
+  version: 3.8
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

Comment:
I guess this means `python -m pip install .[docs]`? It's not clear from the docs page.

Reply:
Yep. I think there's an  example in the RtD  docs that makes this somewhat clear, but you're right  that it doesn't seem  to be spelled out explicitly.

Reply:
For reference, I was looking at the bit just after the text "With the previous settings, Read the Docs will execute the next commands:"